### ISSUE TITLE
version 0.0.38

### ISSUE THREAD

**Issue by Valemax1981** [CVMEventi/Countdown#28]
Hi there in version 0.0.38 the countdown windows not remain on top of other windows

**Comment by cipi1965** [comment]
Hi, sorry for that. Tomorrow i'll check it

**Comment by cipi1965** [comment]
Hello, i did try the setting and it works, could you try toggle it?

**Comment by Valemax1981** [comment]
Yes, i tried and nothing changes... As example powerpoint slideshow now override the countdown

**Comment by cipi1965** [comment]
Maybe is Windows specific, i'm trying on macOS, give me some more time

**Comment by cipi1965** [comment]
Should be fixed in v1.0
